src/main/flight/pid_luxfloat.c

```diff
@@ -85,8 +85,13 @@ STATIC_UNIT_TESTED int16_t pidLuxFloatCore(int axis, const pidProfile_t *pidProf
     // -----calculate P component
     float PTerm = luxPTermScale * rateError * pidProfile->P8[axis] * PIDweight[axis] / 100;
     // Constrain YAW by yaw_p_limit value if not servo driven, in that case servolimits apply
-    if (axis == YAW && pidProfile->yaw_p_limit && motorCount >= 4) {
-        PTerm = constrainf(PTerm, -pidProfile->yaw_p_limit, pidProfile->yaw_p_limit);
+    if (axis == YAW) {
+        if (pidProfile->yaw_lpf) {
+            PTerm = filterApplyPt1(PTerm, &yawFilterState, pidProfile->yaw_lpf, dT);
+        }
+        if (pidProfile->yaw_p_limit && motorCount >= 4) {
```

Comment:
just reviewing this, and I'm trying to work out why constraining is not applied to models with less than 4 motors.  shouldn't this always be calculated?